### PR TITLE
static advection-diffusion equation example - first commit

### PR DESCRIPTION
**Pull request by kmith**

### CODE REVIEW

File: src/fortran/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (hsorby) [inline src/fortran/.gitignore]: This file is not required.  A .gitignore file should be place in the root directory of a repository in any case.

**Review by hsorby — CHANGES_REQUESTED** [review]
Looks good.  Remove the empty .gitignore file (which is in the wrong location) and we can merge this in.

File: src/fortran/.gitignore (hunk 1 above)
Comment (kmith) [reply src/fortran/.gitignore]: Removed .gitignore in src/fortran.

**Comment by kmith** [comment]
Removed .gitignore in src/fortran.

**Pull request merged**